lib/simp/cli/commands/bootstrap.rb

```diff
@@ -155,91 +91,133 @@ def self.run(args = [])
 
     # Print intro
     system('clear')
-    puts
-    puts "*** Starting SIMP Bootstrap ***"
-    puts "   If this runs quickly, something wrong happened. To debug the problem,"
-    puts "   run 'puppet agent --test' by hand or read the log. The log can be found"
-    puts "   at '#{@logfile.path}'."
-    puts
+    say ("<%= color('=== Starting SIMP Bootstrap ===', :yellow) %>")
+
+    # Set an interrupt trap if safe mode is enabled
+    say "> The log can be found at '#{@logfile.path}'\n"
+    if not @unsafe
+      Signal.trap("INT") { say "<%= color('Safe mode enabled, ignoring interrupt', :magenta) %>"}
+      say "> INFO: SAFE mode enabled" if @verbose
+    else
+      say "> INFO: SAFE mode disabled" if @verbose
+    end
+
 
     # Kill all puppet processes and stop specific services
-    puts "Killing all Puppet processes, httpd and removing Puppet ssl certs.\n\n" if @verbose
-    system("pkill -9 -f puppetmasterd >& /dev/null")
+    say "> <%= color('Killing all Puppet processes', :cyan) %>"
     system("pkill -9 -f puppet >& /dev/null")
     system('pkill -f pserver_tmp')
     system("puppet resource service puppetserver ensure=stopped >& /dev/null")
-    system("puppet resource service httpd ensure=stopped >& /dev/null")
-    FileUtils.rm_rf(Dir.glob(File.join(::Utils.puppet_info[:config]['ssldir'],'*')))
-    FileUtils.rm_f(Dir.glob(File.join(::Utils.puppet_info[:config]['rundir'],'*')))
-    FileUtils.touch('/.autorelabel')
-
-    puts "*** Starting the Puppetmaster ***"
-    puts
 
-    FileUtils.mkdir_p("#{::Utils.puppet_info[:config]['vardir']}/pserver_tmp")
-    FileUtils.chown('puppet','puppet',"#{::Utils.puppet_info[:config]['vardir']}/pserver_tmp")
-    system(%{puppet resource simp_file_line puppetserver path='/etc/sysconfig/puppetserver' match='^JAVA_ARGS' line='JAVA_ARGS="-Xms2g -Xmx2g -XX:MaxPermSize=256m -Djava.io.tmpdir=#{::Utils.puppet_info[:config]['vardir']}/pserver_tmp"' 2>&1 > /dev/null})
-
-    if File.directory?('/etc/puppetlabs/puppetserver/conf.d')
-      puppetserver_dir = '/etc/puppetlabs/puppetserver/conf.d'
+    # Kill the connection with puppetdb
+    say "> <%= color('Killing connection to PuppetDB', :cyan) %>"
+    system("puppet resource service puppetdb ensure=stopped >& /dev/null")
+    confdir = ::Utils.puppet_info[:config]['confdir']
+    if File.exists?("#{confdir}/routes.yaml")
+      system("rm -f #{confdir}/routes.yaml")
+      say "> INFO: <%= color('Successfully removed #{confdir}/routes.yaml', :green) %>" if @verbose
```

Comment:
anything with @verbose at the end looks like  DEBUG: message not an INFO: message